### PR TITLE
Add tools/scan_symbols.py script

### PR DESCRIPTION
**Pull request by wmww**
Given the path to a .so file, this script will compare the symbols in the library with the symbols in the debian symbols file and show the symbols only present in one or the other. Not perfect, but makes manually editing symbols easier.

### CODE REVIEW

File: tools/scan_symbols.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```
Comment (AlanGriffiths) [inline tools/scan_symbols.py:1]: This doesn't work for me - Ubuntu doesn't interpret `python` as `python3`

FWIW other scripts are using:

```suggestion
#! /usr/bin/python3
```
And that does work

Comment (wmww) [reply tools/scan_symbols.py]: Huh, my 21.10 system does. I wonder if I explicitly set that up at some point.

**Review by AlanGriffiths — APPROVED** [review]
Probably useful and certainly harmless

bors merge

File: tools/scan_symbols.py (hunk 1 above)
Comment (AlanGriffiths) [reply tools/scan_symbols.py]: 21.10 is EOL!

**Pull request merged**